src/hooks/use-snapshot.ts

```diff
@@ -12,38 +12,27 @@ interface UseSnapshotProps {
 }
 
 const useSnapshot = ({ layoutService, viewService, rect, model }: UseSnapshotProps): stardust.Rect => {
-  onTakeSnapshot(async (copyOfLayout: SnapshotLayout) => {
-    if (!copyOfLayout.snapshotData) {
-      return copyOfLayout;
+  onTakeSnapshot(async (snapshotLayout: SnapshotLayout) => {
+    if (!snapshotLayout.snapshotData || !model || snapshotLayout.snapshotData.content) {
```

Comment:
Is the `content` object always there?

Reply:
Just did some quick testing and it does seem like there is no `content` the first time `onTakeSnapshot` is called

Reply:
Just had another look. You are indeed correct 👍